ui-admin/src/study/surveys/FormOptionsModal.tsx

```diff
@@ -111,6 +133,48 @@ export const FormOptions = ({ studyEnvContext, initialWorkingForm, updateWorking
                 })}
               /> Allow study staff to edit participant responses
             </label>}
+            <label className="form-label d-flex align-items-center mt-4">
+              <label className="pe-2">
+                <input type="checkbox" checked={!!workingForm.daysAfterEligible}
+                  onChange={e => updateWorkingForm({
+                    ...workingForm,
+                    daysAfterEligible: e.target.checked ? 365 : undefined
+                  })}
+                /> Delay assigning this survey
+              </label>
+              <label className="d-flex align-items-center">
+                <TextInput value={workingForm.daysAfterEligible} type="number" min={1} max={9999}
+                  className="mx-2"
+                  onChange={val => updateWorkingForm({
+                    ...workingForm, daysAfterEligible: parseInt(val)
+                  })}
+                /> <span className="text-nowrap">days after enrollment</span>
+              </label>
+            </label>
+            <div className="d-flex align-items-center mb-4">
+              <label className="pe-2" htmlFor={selectInputId}>
+                Recurrence:
+              </label>
+              <Select inputId={selectInputId}
+                styles={{
+                  control: baseStyles => ({
+                    ...baseStyles,
+                    minWidth: '13em'
+                  })
+                }}
+                options={options} onChange={onChange}
+                value={selectedOption}/>
+              <label className="d-flex align-items-center ms-3">
+                every <TextInput value={workingForm.recurrenceIntervalDays} type="number" min={1} max={9999}
+                  className="mx-2"
+                  onChange={val => updateWorkingForm({
```

Comment:
optional UX nit: disable the interval input unless relevant
```suggestion
                  disabled={workingForm.recurrenceType === 'NONE'}
                  onChange={val => updateWorkingForm({
```

Reply:
good topic for demo, I kind of like being able to type in it, and just auto-switching the mode when they do